app/screens/AppNavigator/screens/Dex/DexNavigator.tsx

```diff
@@ -19,6 +22,11 @@ export function DexNavigator (): JSX.Element {
         component={DexScreen}
         options={{ headerTitle: translate('screens/DexScreen', 'Decentralized Exchange') }}
       />
+      <DexStack.Screen
+        name='PoolSwap'
+        component={PoolSwapScreen}
+        options={{ headerTitle: translate('screens/DexScreen', 'Decentralized Exchange') }}
```

Comment:
hide back title for ios

Reply:
ok I'll add. But why we need to have it?

Reply:
on iOS, the title will shift automatically depending on the size of the back title and title. It will cause clipping if you use alternative header, hiding it just makes it more consistent across screens